src/Pages/List/Availables/Availables.scss

```
@@ -0,0 +1,113 @@
+.Availables {
+  .rooms {
+    display: flex;
+    padding: 20px 0;
+    // margin-bottom: 30px;
+    border-top: 1px solid rgb(202, 202, 202);
+    // justify-content: space-between;
+
+    .listImgSlider {
+      width: 300px;
+      height: 200px;
+      padding-right: 20px;
+    }
+
+    .infoBox {
+      padding: 0 20px;
+      width: 100%;
+      .description {
+        margin-bottom: 8px;
```

```suggestion
    .infoBox {
      padding: 0 20px;
      width: 100%;
      
      .description {
        margin-bottom: 8px;
```
새로운 셀렉터 앞에서는 한줄 띄워주세요.

---

@tjohnny93 접혀서 숨겨져 있는 코멘트도 보셨나요?